Mage.Sets/src/mage/cards/c/CloseQuarters.java

```diff
@@ -9,6 +9,7 @@
 import mage.cards.CardSetInfo;
 import mage.constants.CardType;
 import mage.constants.TargetController;
+import mage.filter.StaticFilters;
```

Comment:
You've added the import to this file but haven't done anything else.

Reply:
I forgot to remove the import, BecomesBlockedAllTriggeredAbility doesn't accept a FilterControlledCreaturePermanent.

Reply:
Right, I've run into that myself when static-izing filters. Unfortunately `FilterControlledCreaturePermanent` can't be a subclass of both `FilterControlledPermanent` and `FilterCreaturePermanent`.